redpanda/templates/statefulset.yaml

```diff
@@ -101,8 +101,8 @@ spec:
               redpanda
               start
               --smp={{ .Values.statefulset.resources.limits.cpu }}
```

Comment:
Ahh... this would also need to be changed or removed if this comment is approved: https://github.com/redpanda-data/helm-charts/pull/50/files#r869441863

Reply:
The smp is tight to cpu. It is recommended to have 2G per one smp (core), but with the `overprovisioned` flag set to `true` the strict restriction is removed. 

ref: https://github.com/redpanda-data/helm-charts/blob/74d3d106987b9aefd087f2654f39473bd24019b6/redpanda/values.yaml#L107